lib/aruba/environment/windows_environment.rb

```diff
@@ -0,0 +1,11 @@
+require 'aruba/environment/basic_environment'
+
+module Aruba
+  class WindowsEnvironment < BasicEnvironment
+    def initialize
```

Comment:
> ut I didn't know if changes to ENV would be 'rolled back' after the program ends. 

They are local to that sub-process. So no problem here. Let's try my version first. If someone complains, we will change that.

> Are you asking 'is this the approach that all programs must take when they're dealing with Windows ENVs?' the answer is yes. (But I don't think that's what you're asking.)

Yes, this was my question. :-) Should work on my English. Just try my suggestion first.